.github/workflows/build.yml

```diff
@@ -6,7 +6,7 @@ on:
             - "*"
         paths-ignore:
             - ".gitignore"
-    pull_request:
+    pull_request_target:
```

Comment:
I seem to remember this being `pull_request` for a reason - maybe it was so that `detekt` would run off of the branch code instead of the repo code?